### ISSUE TITLE
a11y_SkipLink> Default> A11y> 1 violations is appearing

### ISSUE THREAD

**Issue by priya-rajput** [cengage/react-magma#515]
**Describe the bug**
SkipLink>> "1 violations" is appearing for "Default". 

**To Reproduce**
1. Go to 'https://storybook-preview-290--upbeat-sinoussi-f675aa.netlify.app/?path=/story/skiplink--default'
2. Click on 'Accessibility" add on
3. "1 violations" is appearing

**Expected behavior**
"0 violations" should appear for SkipLink> Default.

**Screenshots**
https://somup.com/c3n26nTccd

**Desktop :**
 - OS: [Win 10]
 - Browser [chrome]
 - Version [Version 98.0.4758.82 (Official Build) (64-bit)]

**Comment by Uditsingh67** [comment]
The issue is fixed.
Storybook Link : https://storybook-preview-dev--upbeat-sinoussi-f675aa.netlify.app/?path=/story/skiplink--default
See Screen-cast : http://somup.com/c3jQeAuJTt
Thanks!

**Comment by Uditsingh67** [comment]
issue does not exist.
See Screen-cast : http://somup.com/c3jviZUlHF
